lib/srv/transport/transportv1/transport.go

```diff
@@ -308,9 +324,17 @@ func (s *Service) ProxySSH(stream transportv1pb.TransportService_ProxySSHServer)
 
 	// monitor the user connection
 	conn := streamutils.NewConn(sshStreamRW, p.Addr, targetAddr)
-	monitorCtx, userConn, err := s.cfg.ConnectionMonitor.MonitorConn(ctx, authzContext, conn)
-	if err != nil {
-		return trace.Wrap(err)
+	var monitorCtx context.Context
+	var userConn net.Conn
+	if unscopedCtx, ok := authzContext.UnscopedContext(); ok {
+		monitorCtx, userConn, err = s.cfg.ConnectionMonitor.MonitorConn(ctx, unscopedCtx, conn)
+		if err != nil {
+			return trace.Wrap(err)
+		}
+	} else {
+		// TODO(fspmarshall/scopes): implement connection monitoring for scoped identities
+		userConn = conn
+		monitorCtx = ctx
```

Comment:
Is this safe? At a glance it doesn't look like `MonitorConn` ever returns the original context outside of error cases, so I'm wondering if there are potentially bad downstream effects from simply forwarding the conn/ctx for scoped identities?